libs/test-common/src/main/java/org/codice/ddf/test/common/configurators/KarafOptions.java

```diff
@@ -82,6 +85,7 @@ public Option get() {
 
   private Option getVmOptions() {
     return composite(
+        java9VmOptions(),
```

Comment:
❓ Okay this is probably fine but I'll ask anyway - we're not going to get in a weird situation where production is running on Java 11 but our pax exam version is running on Java 9, are we? That would be bad. 

I'm assuming this `java9VmOptions()` is equally viable and needs to be called on both the J9 and J11 situations. I'm assuming nothing changed between version 9 and version 11 to warrant another function to be written by pax exam. Correct?

Reply:
I called this Java 9 VM options since Java 9 is the version that introduced Jigsaw and the module system.